evergreen/config.yml

```diff
@@ -59,9 +59,18 @@ tasks:
                     -j ${max_jobs|$(grep -c proc /proc/cpuinfo)}
         fi
 
+        if [ -n "${run_tests_against_baas|}" ]; then
+            set_cmake_var baas_vars REALM_ENABLE_AUTH_TESTS BOOL On
+            set_cmake_var baas_vars REALM_MONGODB_ENDPOINT STRING "http://localhost:9090"
+            set_cmake_var baas_vars REALM_STITCH_CONFIG PATH $(pwd)/test/object-store/mongodb/stitch.json
+        fi
+
         set_cmake_var realm_vars REALM_BUILD_COMMANDLINE_TOOLS BOOL On
         set_cmake_var realm_vars REALM_ENABLE_SYNC BOOL On
         set_cmake_var realm_vars REALM_ENABLE_ENCRYPTION BOOL On
+        if [[ -n "${fetch_missing_dependencies|}" ]]; then
+            set_cmake_var realm_vars REALM_FETCH_MISSING_DEPENDENCIES BOOL On
+        fi
```

Comment:
Isn't this identical to the next block?

Reply:
Oh, yup. I think this is a rebase issue. Will fix.